content/en/tracing/api_catalog/_index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Datadog API Catalog
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+---
+
+{{< beta-callout url="https://datadoghq.com" d-toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger">}}
+  Datadog API Catalog is in beta! Use this form request access. 
+{{< /beta-callout >}} 
+
+{{< img src="tracing/api_catalog/api-catalog-overview.png" alt="API Catalog showing a list of API endpoints along with their Error Rate, P99 Latency, Requests Count, and Owning Team, with searching, filtering, and scoping features." style="width:100%;" >}}
```

Comment:
see a new better  image here - https://a.cl.ly/12uRq4jL

(we have the sorting working so it is easier to get a better view of the Catalog